File: pgm/ABC_Visual_Basic/ABC_164_B.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim A, B, C, D As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1) : C = arr(2): D = arr(3)
        Console.WriteLine(
            If(Math.Floor((B + C - 1) / B) <= Math.Floor((A + D - 1) / D) , "Yes", "No"))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_164_B.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int A, B, C, D;
        A = int.Parse(arr[0]);
        B = int.Parse(arr[1]);
        C = int.Parse(arr[2]);
        D = int.Parse(arr[3]);
        Console.WriteLine((B + C - 1) / B <= (A + D - 1) / D ? "Yes" : "No");
        return;
    }
}